lib/resources/geo-extracts.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2025 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const { getOrNotFound } = require('../util/promise');
+const { Form } = require('../model/frames');
+const { Sanitize } = require('../util/param-sanitize');
+
+
+module.exports = (service, endpoint) => {
+
+  service.get('/projects/:projectId/forms/:formId/submissions.geojson', endpoint(async ({ Forms, GeoExtracts }, { auth, query, params }) => {
+
+    const form = await Forms.getByProjectAndXmlFormId(params.projectId, params.formId, Form.WithoutDef, Form.WithoutXml)
+      .then(getOrNotFound)
+      .then((foundForm) => auth.canOrReject('submission.read', foundForm));
```

Comment:
Just to follow up on https://github.com/getodk/central-backend/pull/1555#discussion_r2253165469, I still feel like `submission.list` would be better here than `submission.read`. Either way is fine though. We use `submission.read` almost everywhere, including OData, even for endpoints where I think `submission.list` would be more appropriate. I've filed the issue #1326 to track this thought. I think it'd be fine to use `submission.list` here now, with the thought that it's the more correct verb; it'd also be fine to leave it as-is and come back to it later as part of #1326.